american-journal-of-climate-change.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Climate Change</title>
+    <title-short>AJCC</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-climate-change</id>
+	<link href="http://www.zotero.org/styles/american-journal-of-climate-change" rel="self"/>
+	<issn>2167-9495</issn>
+      <eissn>2167-9509</eissn>
+	<link href="http://www.scirp.org/journal/ajcc/" rel="documentation"/>
+      <contributor>
+      <name>Arunkumar R</name>
+      <uri>http://www.mendeley.com/profiles/arunkumar-r1/</uri>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
```

Comment:
please use something less generic ;). Full list of options is here: http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories
I'd guess `science` and/or `geology` would work.